### PR TITLE
Add min_lr parameter to ReduceLROnPlateauCallback

### PR DESCRIPTION
**Pull request by rohitgr7**
Add `min_lr` for lower bound on learning rate in `ReduceLROnPlateauCallback`.

### CODE REVIEW

File: fastai/callbacks/tracker.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def on_epoch_end(self, epoch, **kwargs:Any)->None:
             self.best,self.wait = current,0
         else:
             self.wait += 1
-            if self.wait > self.patience:
+            if self.wait >= self.patience:
```
Comment (sgugger) [inline fastai/callbacks/tracker.py]: I wouldn't change this behavior as a. it's the one users know, b. it's consistent with other callbacks. We can clarify in the documentation it takes patience full epochs before applying the change at patience+1.

File: fastai/callbacks/tracker.py [hunk 2]
```diff
@@ -124,7 +124,7 @@ def on_epoch_end(self, epoch, **kwargs:Any)->None:
         if self.operator(current - self.min_delta, self.best): self.best,self.wait = current,0
         else:
             self.wait += 1
-            if self.wait > self.patience:
+            if self.wait >= self.patience and self.opt.lr > self.min_lr:
```
Comment (sgugger) [inline fastai/callbacks/tracker.py]: Same here

**Comment by sgugger** [comment]
Thanks for your PR. I don't think we should change the behavior with patience though.

File: fastai/callbacks/tracker.py (hunk 1 above)
Comment (rohitgr7) [reply fastai/callbacks/tracker.py]: Changed as suggested.

**Comment by sgugger** [comment]
Thanks!

**Pull request merged**